exercises/practice/automated-readability-index/.docs/instructions.md

```diff
@@ -0,0 +1,112 @@
+# Instructions
+
+As we grow up, we can understand more and more complex texts. But how is the level of complexity of a given text assessed, and how can we make programs which do that? 
+In this exercise, you will write a program which will determine how difficult a text is to read
+and what age it is best suited for.
+
+We will use the
+[Automated readability index (ARI)][wiki] formula for calculating the text complexity.
+The ARI was developed in 1967 and assesses the U.S. grade level required to read 
+a piece of text. In some ways, it is similar to other formulas. Its main difference 
+is rather than counting syllables, it counts characters. The more characters per word, 
+the harder the word. It also counts the number of words per sentences. This sets it apart from some 
```

Comment:
Presumable, "the more words per sentence, the harder the sentence"?
Maybe combine both sentences? Something along the lines of, "It counts the number of characters per word and words per sentence. The higher the count, the more complex the text is considered."

Reply:
I copied this piece of text from one article. Because English is not my native language, I am not very good at writing it. If it is possible to rewrite this fragment with other words, then I think this should be done.

Reply:
If I were entirely rewriting it, I might suggest something like:

> We will use the [Automated readability index (ARI)][wiki] formula for calculating the text complexity.
> The ARI was developed in 1967 and assesses the U.S. grade level required to read 
a piece of text. It does this by measuring the complexity of words and sentences. Complexity is measured as characters per word and words per sentence. A higher value is more complex.

How does that sound?

Reply:
It sounds good to me. 
Do we need `the` before `text` in the first sentence?

Reply:
fixed